FAQ.md

```diff
@@ -0,0 +1,18 @@
+# Frequently Asked Questions
+
+### How do I configure DOM events with flags like `preventDefault` and `stopPropagation`?
+
+Using `element`, a low-level DOM builder function. Its second argument, `ElementConfig`, has a `_elementConfig_eventSpec` field that can be modified using `addEventSpecFlags`. It takes a `Proxy` that specifies the `DomSpace`, an event name, and a callback which takes the event result and returns an event flag, which can enable behavior like `preventDefault` and `stopPropagation`.
+
+For example:
+
+```haskell
+link :: forall t m a. DomBuilder t m => m a -> m (Event t (), a)
+link c = do
+  let cfg = (def :: ElementConfig EventResult t (DomBuilderSpace m))
```

Comment:
The documentation should explain why this type signature is necessary.

Reply:
Okay, just added a justification for the type signature. It seems like a general type inference problem, so I didn't go much into how it specifically applies here, but I think it provides enough for anyone interested in learning more.